packages/core/js-sdk/src/auth/index.ts

```diff
@@ -0,0 +1,33 @@
+import { Client } from "../client"
+import { Config } from "../types"
+
+export class Auth {
+  private client: Client
+  private config: Config
+
+  constructor(client: Client, config: Config) {
+    this.client = client
+    this.config = config
+  }
+
+  login = async (payload: { email: string; password: string }) => {
```

Comment:
thought: this doesn't seem to support other provider logins. 

would expect a signature like this `authenticate("scope", "provider", { ...providerData })`

Reply:
@srindom yeah I think the exact interface needs to be finalized (as you can see, even the typings are hard-coded for now. I just wanted to replace what we had in admin to put the SDK in use. I will be doing more improvements today, and probably in the following week.